zetaclient/zetacore/query.go

```diff
@@ -266,6 +283,7 @@ func (c *Client) GetLastBlockHeightByChain(chain chains.Chain) (*crosschaintypes
 	return resp.LastBlockHeight, nil
 }
 
+// GetBlockHeight returns the block height
```

Comment:
```suggestion
// GetBlockHeight returns the zetachain block height
```

Reply:
Maybe we can rename to `GetLatestZetaBlockHeight()` . so that its similar to `GetZetaBlock()`